Cesium.CodeGen/Ir/TopLevel/TopLevelDeclaration.cs

```diff
@@ -88,7 +88,7 @@ public void EmitTo(TranslationUnitContext context)
         {
             var field = context.AssemblyContext.ResolveGlobalField(name, context);
             var globalInitializerScope = context.GetInitializerScope();
-            initializer.Lower(globalInitializerScope).EmitTo(globalInitializerScope);
+            initializer.Lower((IDeclarationScope)globalInitializerScope).EmitTo(globalInitializerScope);
```

Comment:
I suggest getting rid of this cast by making `context.GetInitializerScope()` to return a `GlobalConstructorScope`. It's internal, anyway, so not a big deal.